src/showingpreviously/cinemas/birks_cinema.py

```diff
@@ -0,0 +1,115 @@
+import re
+import bs4
+import datetime
+
+import showingpreviously.requests as requests
+from showingpreviously.model import ChainArchiver, CinemaArchiverException, Chain, Cinema, Screen, Film, Showing
+from showingpreviously.consts import UK_TIMEZONE, UNKNOWN_FILM_YEAR
+
+BASE_URL = 'https://birkscinema.co.uk'
+FILM_SCHEDULE_URL = 'https://birkscinema.co.uk/whats-on/on-screen/#dates'
+FILM_NAME_IGNORES = []
+
+CHAIN = Chain('The Birks Cinema')
+CINEMA = Cinema('The Birks Cinema', UK_TIMEZONE)
+SCREEN = Screen('Screen 1')
+
+film_year_cache = {}
+
+
+def get_response(url: str) -> requests.Response:
+    r = requests.get(url)
+    if r.status_code != 200:
+        raise CinemaArchiverException(f'Got status code {r.status_code} when fetching URL {url}')
+    return r
+
+
+def format_film_name(film_name: str) -> (str, dict[str, any]):
+    attributes = {'format': []}
+    # convert to lowercase for easier processing
+    film_name = film_name.lower()
+    # check to see if this is a live event
+    if film_name.startswith('national theatre live: '):
```

Comment:
It looks like Birks also does `relaxed screenings` (eg. for [White Christmas](https://birkscinema.co.uk/whats-on/on-screen/white-christmas-relaxed-screening/)) - good to have this as an attribute maybe? Or at least strip it out of the title.